{"name": "test_create_tmpfile", "status": "broken", "statusDetails": {"message": "IsADirectoryError: [Errno 21] Is a directory: './../tmp/'", "trace": "test_convert_yml_to_dict = {'Connection': {'DUT1': {'interfaces': [{'LocalPort': 'Ethernet1', 'RemoteDevice': 'DUT2', 'RemotePort': 'Ethernet11'}...'access': {'ip': '10.1.1.2', 'port': 22, 'protocol': 'ssh'}, 'credentials': {'password': 'onl', 'username': 'admin'}}}}\n\n    def test_create_tmpfile(test_convert_yml_to_dict):\n        global new_filename\n        temp = test_convert_yml_to_dict\n        temp1 = './../tmp/'\n        new_filename = temp1 + new_filename\n    \n        print(new_filename)\n    \n>       with open(new_filename, 'w+') as f:\nE       IsADirectoryError: [Errno 21] Is a directory: './../tmp/'\n\ntest_4.py:62: IsADirectoryError"}, "attachments": [{"name": "stdout", "source": "1bb54e5a-925f-42b2-b1ef-520a2a5b19b7-attachment.txt", "type": "text/plain"}], "start": 1656014077468, "stop": 1656014077468, "uuid": "c033554f-7a48-405f-a1cf-d1e073e33f49", "historyId": "047e174eb21cd35aba20cf61d809f7c0", "testCaseId": "4beb7ac807856f851461fca2a912afa6", "fullName": "test_4#test_create_tmpfile", "labels": [{"name": "suite", "value": "test_4"}, {"name": "host", "value": "alexander-Virtual-Machine"}, {"name": "thread", "value": "7997-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_4"}]}